File: GoTravel/src/main/java/com/gotravel/entity/Usuario.java
package com.gotravel.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;


@Entity
@Table(name="Usuario")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Usuario {

    @Id
    @Column(name="idUsuario")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int idUsuario;

    @Column(name="nombre")
    private String nombre;

    @Column(name="email")
    private String email;

    @Column(name="password")
    private String password;

    public Usuario(String nombre, String email, String password) {
        this.nombre = nombre;
        this.email = email;
        this.password = password;
    }

}

File: ClienteMovil/GoTravel/app/src/main/java/com/gotravel/clienteMovil/data/Usuario.kt
package com.gotravel.clienteMovil.data

import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter


@Entity(tableName = "Usuario")
@Serializable
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
class Usuario (

    @PrimaryKey
    @SerialName(value = "idUsuario")
    private val idUsuario: Int,
    @SerialName(value = "nombre")
    private var nombre: String,
    @SerialName(value = "email")
    private var email: String,
    @SerialName(value = "password")
    private var password: String,

)